MCprep_addon/spawner/spawn_util.py

```diff
@@ -813,72 +813,72 @@ def draw_item(self, context, layout, data, set, icon, active_data, active_propna
 
 class ListMobAssetsAll(bpy.types.PropertyGroup):
 	"""For listing hidden group of all mobs, regardless of category"""
-	description = bpy.props.StringProperty()
-	category = bpy.props.StringProperty()
-	mcmob_type = bpy.props.StringProperty()
-	index = bpy.props.IntProperty(min=0, default=0)  # for icon drawing
+	description: bpy.props.StringProperty()
+	category: bpy.props.StringProperty()
+	mcmob_type: bpy.props.StringProperty()
+	index: bpy.props.IntProperty(min=0, default=0)  # for icon drawing
```

Comment:
In this file here and below: If we want to merge this to dev now, then we should revert the use of annotations. Use of annotations and typing should go into the (not yet existing) `MCprep_v350` feature branch instead. Otherwise it would be an immediate drop of support for Blender 2.7x.

Reply:
I'm planning on holding off until 3.5 to merge this